.github/workflows/self-test.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: Self-Test
+
+on: # NOLINT
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Statick
```

Comment:
Nit: I like to put a blank line between each step for readability.

Reply:
I did orginally, but then I believe statick told me I had "more than 3" blank lines which was somehow too many, so I got rid of them all.   I can experiement with adding blank lines back in.